Makefile

```
@@ -13,21 +13,29 @@ extract:
 	docker compose run extractors uv run python main.py configs/arbitrum_mainnet.yaml
 	docker compose run extractors uv run python main.py configs/arbitrum_sepolia.yaml
 
-index:
-	docker compose run indexer --network_name base_mainnet --protocol_name synthetix
-	docker compose run indexer --network_name arbitrum_mainnet --protocol_name synthetix
+listener-start:
+	docker compose up -d indexer-listener
+
+listener-stop:
+	docker compose down indexer-listener
+
+index-start:
+	docker compose -f docker-compose.indexers.yml up -d
+
+index-stop:
+	docker compose -f docker-compose.indexers.yml down 
 
 synths:
 	docker compose run transformer python scripts/get_synths.py
```

```suggestion
	docker compose run transformer uv run python scripts/get_synths.py
```
think we can update this while we're at it